packages/components/src/components/Callout/Callout.tsx

```diff
@@ -126,7 +127,6 @@ const Callout = React.forwardRef<HTMLDivElement, CalloutProps>(
         gap="space40"
         padding="space50"
         ref={ref}
-        role="alert"
```

Comment:
Why remove this?

Reply:
Because a callout is not an alert. The alert is a role we use when something appears on the screen, and we need the user to immediately look into the content that just appeared.